DataFormats/Phase2L1Taus/src/classes.h

```diff
@@ -0,0 +1,14 @@
+#include "Rtypes.h"
+
+#include "DataFormats/Phase2L1Taus/interface/L1HPSPFTau.h"
+#include "DataFormats/Phase2L1Taus/interface/L1HPSPFTauFwd.h"
+
+namespace DataFormats_Phase2L1Taus
+{
+  struct dictionary 
+  {
+    l1t::L1HPSPFTau l1hpspftau;
+    l1t::L1HPSPFTauCollection l1hpspftauCollection;
+    edm::Wrapper<l1t::L1HPSPFTauCollection> l1hpspftauCWrapper;
```

Comment:
I'm still not 100% sure, but I think these could be omitted.

Reply:
Correct, nowadays only the `#include`s are needed in `classes.h`.